### PR TITLE
Fe homepage layout

### PR DESCRIPTION
**Pull request by MilenaHtec**

### CODE REVIEW

File: frontend/src/Components/Profile/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import "./Profile.css";
+import { IconContext } from "react-icons";
+
+const Profile = () => {
+   const [sidebar, setSidebar] = useState(false);
+
+   const showSidebar = () => setSidebar(!sidebar);
+
+   return (
+      <>
```
Comment (marija-antic) [inline frontend/src/Components/Profile/Profile.jsx]: No need for fragment.

**Review by marija-antic — COMMENTED** [review]
I had a couple of comments on previous PR that applies to this one also.

File: frontend/src/Components/Profile/Profile.jsx (hunk 1 above)
Comment (MilenaHtec) [reply frontend/src/Components/Profile/Profile.jsx]: Yes, there is a container. Tnx

File: frontend/src/Components/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                  </Link>
+               </li>
+
+               <li className="nav-text">
+                  <Link to="/">
+                     <AiIcons.AiFillHome />
+                     <span>Home</span>
+                  </Link>
+               </li>
+               <li className="nav-text">
```
Comment (marija-antic) [inline frontend/src/Components/Sidebar/Sidebar.jsx:36]: This kind of <li />. is good candidate for a component, it is repeating.

File: frontend/src/Components/pages/Settings.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                  </div>
+               </div>
+               <div className="col-md-6">
+                  <div className="form-outline mb-1">
+                     <Label id="lastNameInput">Last Name</Label>
+                     <input type="name" name="lastName" id="lastNameInput" className="form-control" required autoComplete="off" />
+                  </div>
+               </div>
+            </div>
+            <div className="form-outline mb-1">
```
Comment (marija-antic) [inline frontend/src/Components/pages/Settings.jsx:27]: Component candidate.

**Pull request closed without merging**